website/docs/downloads/development-builds.md

```diff
@@ -90,10 +90,11 @@ function set_ci_status(workflow_file, os_name, description, page = 1) {
 }
 
 document.addEventListener("DOMContentLoaded", () => {
-    set_ci_status("linux.yml", "linux", "Linux");
-    set_ci_status("macos.yml", "macos", "macOS");
-    set_ci_status("windows-msys2.yml", "msys2", "Windows MSYS2 builds ¹");
-    set_ci_status("windows-msvc.yml", "windows", "Windows MSVC builds ²");
+    set_ci_status("linux.yml", "linux", "Linux x86-64");
+    set_ci_status("platforms.yml", "platforms", "Linux non-x86 ¹");
```

Comment:
Not sure if we want to expose these non-x86 builds on the website at all. AFAIK, these are not "officially" supported by us, they are kind of "nice to haves" because they help making the codebase more robust, etc. They're kinda @kcgen hobby projects within the project, but we won't _really_ want to "officially" support them. 

No support = does not go on the webpage

Reply:
Also, this is on our Linux downloads page:

> Our pre-compiled builds run on most Linux distributions (x86_64 only for now). They depend on the following packages:

So yeah, those other Linux builds are experimental, internal only, etc.

Reply:
Right; the only ARM release package is for macOS. 

Linux and Windows ARM users must get Staging via package manager(s) or build their own from source. 

This is because GitHub doesn't have bonafide CI build hosts for these architectures nor do we have maintainers running these systems.

We still compile aand run unit tests on these though (with Docker and Qemu) because it's valuable feedback and a robustness test of the code, and we do want our code at least compile.

Reply:
>  nor do we have maintainers running these systems.

To me that settles it then. "It builds, unit tests are passing, so ship it", that's not good enough for me 😄 Until we have active maintainers running the builds on these platforms, then we should not add "official" support for them (but keeping the CI builds is a good robustness test and a forward-looking thing yeah; let's definitely keep those).